frontend/src/Component/CollapsibleSection.jsx

```
@@ -0,0 +1,27 @@
+import { useState } from "react";
+import { ChevronDownIcon, ChevronUpIcon } from "@heroicons/react/24/outline";
+
+export default function CollapsibleSection({ title, children }) {
+    const [open, setOpen] = useState(true);
+
+    return (
+        <div className="bg-gray-800 rounded-lg border border-gray-700 shadow my-4">
+            <button
+                onClick={() => setOpen(!open)}
+                className="w-full flex justify-between items-center px-6 py-4 text-left bg-gray-700 hover:bg-gray-600 transition rounded-t-lg"
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)